internal/cmd/root.go

```diff
@@ -0,0 +1,28 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/operator-framework/helm-operator-plugins/internal/cmd/helm-operator/run"
+	"github.com/operator-framework/helm-operator-plugins/internal/version"
+
+	"github.com/spf13/cobra"
+)
+
+func rootCmd() *cobra.Command {
+	rootCmd := &cobra.Command{
+		Use:     "helm-operator-plugins",
+		Short:   "Helm Operator Plugins cli tool",
+		Long:    "A utility that enables the ability to run a helm-based operator",
+		Version: version.Version.String(),
+		Args:    cobra.MinimumNArgs(1),
+	}
+
+	rootCmd.AddCommand(run.NewCmd())
```

Comment:
what's the use of this?

Reply:
Are you asking why root was added? If so `root` was added so this can function like a normal CLI application where a user can run the binary without anything getting executed, to say get the `help` menu. This means there are now sub commands like intended.

Or are you asking what `run.NewCmd` is? This is the sub command for `helm-operator-plugins run` which can be used to run a helm operator. This means in the future, if we wanted to build a container as part of our release process this would be usable.

Reply:
> If so root was added so this can function like a normal CLI application where a user can run the binary without anything getting executed, to say get the help menu

My intention was to understand the use of being able to run this stand alone. I now see this more as a library, with the ability to being able to be re-used in other projects. Which is why having an entry point didn't make a lot of sense. 

This is not a blocker, so I'm fine leaving this as-is.